frontend/src/profiles/EditProfile.js

```diff
@@ -43,30 +46,35 @@ export default function EditProfile() {
             <MenuAppBar pagename="Edit profile"/>
             <MainStyled>
                 <FormStyled onSubmit={handleSubmit}>
+                    {profileData.imageName ? <Avatar alt="profile" src={profileData.imageUrl}/> : <Avatar/>}
+                    <input type="file" onChange={handlePictureChange} />
```

Comment:
with `accept={"image/*"}` only images can be uploaded

```suggestion
                    <input type="file" onChange={handlePictureChange} accept={"image/*"} />
```